improver/calibration/ensemble_calibration.py

```diff
@@ -1583,7 +1583,7 @@ def _calculate_scale_parameter(self) -> ndarray:
         # Calculating the scale parameter, based on the raw variance S^2,
```

Comment:
Does this comment need updating? Maybe to put the square root info in

Reply:
I've updated this comment. I've also made a few other docstring changes in https://github.com/metoppv/improver/pull/1642/commits/d70845133e7ff60880dd47c5f16458f4dbceaf47.